content-src/components/LinkMenuButton/LinkMenuButton.scss

```diff
@@ -6,7 +6,7 @@
   width: $link-menu-button-size;
   height: $link-menu-button-size;
   background-color: $white;
-  background-image: url('img/glyph-more-16.svg');
+  background-image: url($image-path + 'glyph-more-16.svg');
```

Comment:
The inconsistent interpolation is only to workaround sasstools/sass-lint#63 .   I only made the switch to `#{$foo}` in cases where sass-lint was actually falling over on the existing syntax. Agreed that it's probably cleaner to change everything, but I didn't want to do that without more input.  @k88hudson what are your thoughts?